cmd/bee/cmd/start.go

```diff
@@ -151,7 +151,7 @@ func (c *command) initStartCmd() (err error) {
 	}
 
 	cmd.Flags().String(optionNameDataDir, filepath.Join(c.homeDir, ".bee"), "data directory")
-	cmd.Flags().Uint64(optionNameDBCapacity, 5000000, "db capacity in chunks")
+	cmd.Flags().Uint64(optionNameDBCapacity, 5000000, "db capacity in chunks, multiply by 4096 to get bytes")
```

Comment:
It is more correct to say that a chunk is 4104 bytes long in its total size. But the storage also contains indexes and leveldb metadata that would add up. We could state that size in bytes is an approximation and have something less precise as it is currently in swarm cli option.

Reply:
@janos , thanks for comment. 

What about changing it here to `multiply by 4200 to get the approximate capacity in bytes` ?

Reply:
Or `multiply by swarm.ChunkSize to get the approximate capacity in bytes`

Reply:
> Or multiply by swarm.ChunkSize to get the approximate capacity in bytes

Otherwise it does not correspond with the log in `localstore.go:169`

Reply:
Updated